aliasmgr/aliasmgr.go

```diff
@@ -302,8 +303,9 @@ func (m *Manager) AddLocalAlias(alias, baseScid lnwire.ShortChannelID,
 	// Update the aliasToBase and baseToSet maps.
 	m.baseToSet[baseScid] = append(m.baseToSet[baseScid], alias)
 
-	// Only store the gossiper map if gossip is true.
-	if gossip {
+	// Only store the gossiper map if gossip is true, or if the caller
+	// explicitly asked to store this reverse mapping.
+	if gossip || baseLookup {
```

Comment:
is it the desired behaviour that this reverse lookup is not persisted? currently, all these mappings added will be lost on restart